src/snowflake/snowpark/stored_procedure.py

```diff
@@ -408,7 +445,7 @@ class StoredProcedureRegistration:
             -------------
             <BLANKLINE>
 
-    Example 9
+    Example 11
```

Comment:
thank you. I've been meaning to open a small PR to fix this but have been too lazy to do so.